lms/static/sass/course/_student-notes.scss

```diff
@@ -45,6 +45,12 @@ $divider-visual-tertiary: ($baseline/20) solid $gray-l4;
       @extend .content; // needed extend carried over from course handouts UI, but should be cleaned up
       width: 100%;
     }
+
+    .sr-is-focusable,
```

Comment:
Let's remove this from here. I've got another [PR](https://github.com/edx/edx-platform/pull/8045) that places this in a more available place.